docs_headless/astro.config.mjs

```diff
@@ -240,3 +264,19 @@ export default defineConfig({
         assets: 'assets',
     },
 });
+
+/**
+ * @param {string} name
+ * @returns {any}
+ */
+function enterpriseEntry(name) {
+    return {
+        link: name.toLowerCase().replace(/</g, '').replace(/>/g, ''),
+        label: name,
+        attrs: { class: 'enterprise' },
+        badge: {
+            text: 'React Admin Enterprise',
+            variant: 'default',
+        },
```

Comment:
Is the badge used somewhere? I don't see it... :thinking: 

UPDATE: or is it for a11y?